docs/portal/developer-portal/operations/Basic_UAN_Configuration.md

```diff
@@ -64,6 +64,64 @@ CFS uses configuration layers. Configuration layers allow the sharing of Ansible
 #      branch: "{{uan.working_branch}}"
 ```
 
+### Required CFS Layers for UAN Configuration
+
+### Slingshot Host Software (playbook: shs_{{default.network_type}}_install.yml)
+
+The first CFS Layer installs the Slingshot Host Software for the Slingshot network type of the system.  The `default.network_type` is is either `mellanox` or `cassini`.
+
+### COS (playbook: cos-application.yml)
+
+The second CFS Layer runs the following roles from the `cos-config-management` VCS repository. Any configuration changes needed for these roles must be made in the `cos-config-management` `group_vars` or `host_vars` directories.
+
+The following Ansible roles are run during UAN image configuration:
+
+- #### Standard UNIX configuration
+  - cos-config-map
+  - rsyslog
+  - localtime
+  - ntp
+  - limits
+  - kdump
+- #### Allow trust of CSM generated keys for elective passwordless SSH during image customization
+
+  - trust-csm-ssh-keys
+- #### HPE Cray EX system configurations
+
+  - overlay-preload
+- #### Install COS rpms
+
+  - cos-services-install
+- #### DVS/LNET/FS
+
+  - cray_lnet_install
+  - cray_dvs_install
+  - cray_lnet_load
+  - cray_dvs_load
+
+The following Ansible roles are run during UAN post-boot configuration:
+
+- #### Standard UNIX configuration
+
+  - rsyslog
+- #### HPE Cray EX system configurations
+
+  - ca-cert
+  - overlay-preload
+- #### GPU deploy support
+
+  - cray_gpu_deploy
```

Comment:
Let's change those level 4 headers to inline bold to avoid possible rendering issues. Add backticks for the role names too.

```suggestion
- **Standard UNIX configuration**
  - `cos-config-map`
  - `rsyslog`
  - `localtime`
  - `ntp`
  - `limits`
  - `kdump`
  
- **Allow trust of CSM generated keys for elective passwordless SSH during image customization**
  - `trust-csm-ssh-keys`
  
- **HPE Cray EX system configurations**
  - `overlay-preload`
  
- **Install COS rpms**
  - `cos-services-install`
  
- **DVS/LNET/FS**
  - `cray_lnet_install`
  - `cray_dvs_install`
  - `cray_lnet_load`
  - `cray_dvs_load`

The following Ansible roles are run during UAN post-boot configuration:

- **Standard UNIX configuration**
  - `rsyslog`
  
- **HPE Cray EX system configurations**
  - `ca-cert`
  - `overlay-preload`
  
- **GPU deploy support**
  - `cray_gpu_deploy`
```

Reply:
Addressed comments.